x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -58,12 +58,12 @@ const PROVIDERS = Object.freeze({
   providerName: 'cloud-basic',
 });
 
-const getApiKeyFromElasticCloudJsonFile = (): string | undefined => {
+export const getApiKeyFromElasticCloudJsonFile = (environment: string): string | undefined => {
   const userHomeDir = os.homedir();
   try {
     const jsonString = fs.readFileSync(path.join(userHomeDir, '.elastic/cloud.json'), 'utf-8');
     const jsonData = JSON.parse(jsonString);
-    return jsonData.api_key.qa;
+    return environment === 'prod' ? jsonData.api_key.prod : jsonData.api_key.qa;
```

Comment:
nit: What values `environment` can have?

It looks beneficial to define enum or constants for known environments. Something like

```ts
enum Environment {
  Production = 'prod',
  QA = 'qa'
}
```

and use it instead of hardcoded values. WDYT?

Reply:
Sounds better. On it.